src/fhirConfig.ts

```diff
@@ -151,11 +151,30 @@ export interface ProductInfo {
     copyright?: string;
 }
 
+export interface MultiTenancyConfig {
+    /**
+     * Enable multi-tenancy.
+     * This value is always true. To disable multi tenancy, remove the `multiTenancyConfig` altogether.
+     */
+    enableMultiTenancy: true;
+    /**
+     * When enabled, the tenantId is included on the server url to make it unique for each tenant. i.e. `<baseUrl>/tenant/<tenantId>`
```

Comment:
Url length a concern? could turn "tenant" to "t"

Reply:
I like that `tenant/<tenantId>` is more self-explanatory. Also, that's the URL pattern suggested by MEDHOST. 
Shortening to `t` is not very impactful on the overall URL length

`tenantId` is restricted to max 64 chars (router enforces that).  The `baseUrl` is an APIGW url (~60 chars). 

The above numbers are small when compared to any url limits.

https://docs.aws.amazon.com/apigateway/latest/developerguide/limits.html
> Length, in characters, of the URL for an edge-optimized API = 8192

https://stackoverflow.com/questions/417142/what-is-the-maximum-length-of-a-url-in-different-browsers

Reply:
Here at Black Pear we've already implemented multi-tenancy for FWoA (using a wrapper around the components).  We don't use an explicit path in the url to suggest tenancy so our urls appear as:
`<baseUrl>/FOO/Patient/123` (for FOO tenant).
and for the single tenant case we standardise on a tenant of `fhir` so the above example would be:
`<baseUrl>/fhir/Patient/123`
This is similar to the HAPI FHIR approach https://hapifhir.io/hapi-fhir/docs/server_plain/multitenancy.html